apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/model/DexOrderSortBy.java

```diff
@@ -0,0 +1,27 @@
+package com.apollocurrency.aplwallet.apl.exchange.model;
+
+public enum DexOrderSortBy {
+
+    PAIR_RATE("pair_rate"),
+    HEIGHT("db_id");
```

Comment:
Better to rename to DB_ID to lower confusing since height sort will actually perform as following: ORDER BY height ASC/DESC, db_id ASC.